src/Client.ts

```diff
@@ -48,16 +40,12 @@ export default class Client {
 
     loadResources(): void {
         this.auth = new Auth(this);
-        this.users = new Users(this);
-        this.userSubscriptions = new UserSubscriptions(this);
-        this.basket = new Basket(this);
-        this.basketLine = new BasketLine(this);
-        this.subscriptions = new Subscriptions(this);
-        this.subscriptionAddons = new SubscriptionAddons(this);
         this.codes = new Codes(this);
-        this.toybox = new Toybox(this);
         this.products = new Products(this);
+        this.subscriptions = new Subscriptions(this);
         this.wacc = loadWaccResources(this);
+        this.guest = loadGuestResources(this);
+        this.user = new User(this);
```

Comment:
I think we should stick to a convention of singular or plural, as it seems we're slowly mixing the two.

API convention is usually to use a plural for the resource, so would this be able to change to `users`? And the above as `guests`?